package/yast2-ruby-bindings.spec

```diff
@@ -99,6 +99,7 @@ cd -
 
 %files
 %defattr (-, root, root)
+%{yast_ybindir}/y2base
```

Comment:
yes, I plan to add it when approved, so I use correct yast2-core version to conflict with.